src/components/widgets/AntdDynamicFieldSet.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import Form from 'antd/es/form';
+import 'antd/es/form/style';
+
+import Icon from 'antd/es/icon';
+import 'antd/es/icon/style';
+
+import Button from 'antd/es/button';
+import 'antd/es/button/style';
+
+import Input from 'antd/es/input';
+import 'antd/es/input/style';
+
+const FormItem = Form.Item;
+
+export default class DynamicFieldSet extends React.Component {
```

Comment:
Do we actually need to have it as a state component here? Since this is only dealing with props, can we have it as a stateless component?